crates/swc_ecma_parser/src/error.rs

```diff
@@ -684,6 +687,17 @@ impl SyntaxError {
             )
             .into(),
             SyntaxError::SetterParamRequired => "Setter should have exactly one parameter".into(),
+            SyntaxError::DidYouMean {
```

Comment:
This name looks like a bad name for an error.

Reply:
Any naming suggestions?
What about `UnexpectedTokenWithSuggestion`?

Reply:
I think it's a good name

Reply:
Done